test/integration/imageimporter_test.go

```diff
@@ -103,18 +103,9 @@ func TestImageStreamImport(t *testing.T) {
 		t.Fatalf("unexpected responses: %v %#v %#v", err, isi, isi.Status.Import)
 	}
 
-	if isi.Status.Images[0].Image == nil || len(isi.Status.Images[0].Image.DockerImageLayers) == 0 {
+	if isi.Status.Images[0].Image == nil || isi.Status.Images[0].Image.DockerImageMetadata.Size == 0 || len(isi.Status.Images[0].Image.DockerImageLayers) == 0 {
```

Comment:
If it's `TestImageStreamImportFromV2Repository` can you give it a more descriptive name `TestImageStreamImportOfV1ImageFromV2Repository`